components/x11/libXfont/mapfile-vers

```diff
@@ -183,13 +183,9 @@ SUNWprivate {
 	pcfWriteFont;
 	pcfError;
 
-	# "src/bitmap/snfstr.h" - needed by xfs
-	SnfSetFormat;
```

Comment:
I thought the plan was to merge libraries/applications update in any case and wait feedback before merging xorg and drivers.